otel/README.md

```diff
@@ -77,7 +77,7 @@ Everything is ready. Begin running and processing data.
 
 #### OpenTelemetry Collector
 
-See [metadata.csv][5] for a list of metrics provided by this integration.
+See [OTLP Metrics][5] for a list of metrics provided by this integration.
```

Comment:
Hi Kari, 
similarly, could you please help us change line 80 into:

See the [Metrics section][5] for metrics types and [metadata.csv][8] for a list of metrics provided by this integration.

Reply:
So actually for this one, the sentence needs to remain "See metadata.csv for a list of metrics provided by this integration" because that is a "magic sentence" that our docs build uses to generate into the doc a table of metrics from the CSV file.